core/transport/src/main/proto/proto/journal/journal.proto

```diff
@@ -66,6 +66,7 @@ message JournalEntry {
   optional UpdateInodeEntry update_inode = 35;
   optional UpdateInodeDirectoryEntry update_inode_directory = 36;
   optional UpdateInodeFileEntry update_inode_file = 37;
+  optional LoadDirectory load_directory = 53;
```

Comment:
It seems that this is ordered alphabetically. Can you move it to the right location?

Reply:
Will do, thanks!